src/python/pants/option/global_options.py

```diff
@@ -194,8 +201,14 @@ def register_bootstrap_options(cls, register):
                       'pants.backend.codegen.grpcio.python',
                       'pants.backend.codegen.wire.java',
                       'pants.backend.project_info'],
-             help='Load backends from these packages that are already on the path. '
-                  'Add contrib and custom backends to this list.')
+             help='Register tasks from these v1 backends. The backend packages must be present on '
```

Comment:
Backends might be both v1 and v2 for a long time though, right? The help seems to imply that a backend is one or the other.

The case where something might need to be in both lists should probably be acknowledged/explained here (or made less necessary, maybe, by always loading `@rules` for v1 backends).

Reply:
They are indeed one or the other, logically. 

But it is true that a single `register.py` can act as both a v1 and a v2 backend, but they are still two logically distinct entities (e.g., they don't share any entrypoints). 

I tried, evidently unsuccessfully, to make the help reflect this, will try and improve.